apps/nsqd/main.go

```diff
@@ -64,7 +63,9 @@ func (p *program) Start() error {
 	}
 	cfg.Validate()
 
-	options.Resolve(opts, flagSet, cfg)
+	if err := opts.Resolve(flagSet, cfg); err != nil {
```

Comment:
This can be reverted as well after addressing my previous comment. For better or worse, `options.Resolve` fatally exits on errors.